src/lib/components/RelatedWorksField.js

```diff
@@ -26,6 +27,10 @@ export class RelatedWorksField extends Component {
 
     return (
       <>
+        <label className="helptext">
+          Specify identifiers of related works. Supported identifiers include DOI, Handle, ARK, \
+          PURL, ISSN, ISBN, PubMed ID, PubMed Central ID, ADS Bibliographic Code, arXiv, Life Science Identifiers (LSID), EAN-13, ISTC, URNs, and URLs.'
+        </label>
```

Comment:
Looking at the screenshot it seems like some bottom padding on this one would help separate it from the ArrayField, no?